src/axolotl/utils/config.py

```diff
@@ -71,6 +71,11 @@ def normalize_config(cfg):
 
     log_gpu_memory_usage(LOG, "baseline", cfg.device)
 
+    if os.environ.get("WANDB_PROJECT") and len(os.environ.get("WANDB_PROJECT", "")) > 0:
+        cfg.wandb_project = os.environ.get("WANDB_PROJECT")
+
+    if os.environ.get("HUB_MODEL_ID") and len(os.environ.get("HUB_MODEL_ID", "")) > 0:
+        cfg.hub_model_id = os.environ.get("HUB_MODEL_ID")
```

Comment:
FYI: this is used for upcoming work of starting `scripts/finetune.py` and having it run without any custom / run specific / user specific info in the Axolotl config.

Reply:
might be better off in the  `setup_wandb_env_vars` function